src/index.html

```diff
@@ -21,7 +24,482 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__picture"
+        href="#"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO-logo"
+        />
+      </a>
+      <nav>
+        <ul class="header__navlist">
+          <li class="header__navlist-items">
+            <a
+              href="#"
+              class="is-active header__navlist-linkstyle"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
According to the BEM methodology, the class name 'header__navlist-items' should be 'header__navlist-item', as it represents a single item, not multiple items.